oaknational/Oak-Web-Application
# Feat/pupil 707/expired lessons units

## Job Story

**When** a unit, lesson or unit is expired
**I want to show the listItem as expired**
**So that users can’t navigate there**.

## How to test

1. Go to https://deploy-preview-2428--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/english-secondary-year-7-l/units
 - confirm expired units are the same as  https://classroom.thenational.academy/subjects-by-year/year-7/subjects/english
2. Go to https://deploy-preview-2428--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/art-secondary-year-9-l/units/identity-who-i-am-73f9/lessons
- confirm expired lessons are the same as https://classroom.thenational.academy/units/identity-who-i-am-73f9
3. confirm you can't navigate to expire units or lessons

## Screenshots

How it used to look (delete if n/a):
<img width="1079" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/1030540/43e819f2-437b-4815-a73c-c0683b9a62cc">


How it should now look:
<img width="1161" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/1030540/6aece564-0b06-44f4-b88a-70ef8156e6e3">


### Acceptance criteria

- [x]  expired units are displayed as expired
- [x]  expired lessons are displayed as expired
